src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -6703,7 +6659,8 @@ class DrawSketchHandlerExternal: public DrawSketchHandler
 
     virtual void deactivated(ViewProviderSketch *sketchgui)
     {
-        sketchgui->setAxisPickStyle(true);
+        Q_UNUSED(sketchgui);

```

Comment:
```suggestion
            (void) sketchgui;
```
I'm always a bit disturbed when using Qt macro when there is no strong link to Qt (not inside a QObject or other). But maybe that's personal taste only. ;)